branchLayoutImpl/src/main/java/com/virtuslab/branchlayout/impl/BranchLayoutFileParser.java

```diff
@@ -90,8 +90,8 @@ private BaseBranchLayoutEntry createEntry(String line, List<BaseBranchLayoutEntr
    */
   private Array<Tuple2<Integer, Integer>> parseToArrayRepresentation(List<String> lines) throws BranchLayoutException {
     if (lines.size() > 0 && getIndentLevelWidth(lines.head()) > 0) {
-      throw new BranchLayoutException(0, String.format(
```

Comment:
Btw also fixed off-by-one error with the line numbers (the convention literally everywhere is to start from 1 for line numbers)